src/Compilers/CSharp/Test/CommandLine/ErrorLoggerTests.cs

```diff
@@ -120,12 +120,12 @@ public class C
           ""locations"": [
           ],
           ""fullMessage"": ""Program does not contain a static 'Main' method suitable for an entry point"",
+          ""isSuppressedInSource"": false,
           ""properties"": {{
             ""severity"": ""Error"",
             ""defaultSeverity"": ""Error"",
             ""category"": ""Compiler"",
```

Comment:
Note that defaultSeverity, category, etc., are really per-rule metadata. In our SARIF production we will recapitulate this information for each result, true? That seems undesirable. Better would be to produce a ruleInfo array that holds this information.